### DISCUSSION TITLE
Matchmaking

### DISCUSSION THREAD

**Discussion by oppotaku** [ikemen-engine/Ikemen-GO#900] [Feature Request]
You should add matchmaking, i have no idea how you would do that with the current network capability. But it would be nice to matchmake with people instead going to ask someone if theyd like to play. and I also dont know would you would have to do to put characters in matchmaking, cause then people could just cheat, but yeah.

**Comment by Windblade-GR01** [comment]
It would be posible on a full-game environment.
But you will need a matchmaking server for it.
